### PR TITLE
Fix rockcraft and add instructions in CONTRIBUTING

### PR DESCRIPTION
**Pull request by nrobinaubertin**
Fix the permissions of the `/srv/flask` directory inside the rockcraft image.

### CODE REVIEW

File: sample_rock/rockcraft.yaml [hunk 1]
```diff
@@ -38,3 +38,8 @@ parts:
       sample-app: srv/flask/app
     prime:
       - srv/flask/app
+    permissions:
+      - owner: 2000
+        group: 2000
```
Comment (gregory-schiano) [inline sample_rock/rockcraft.yaml:43]: Aren't the permissions kept from the overlay script where there's a chown ?

**Review by gtrkiller — APPROVED** [review]
LGTM

File: sample_rock/rockcraft.yaml (hunk 1 above)
Comment (nrobinaubertin) [reply sample_rock/rockcraft.yaml:43]: The `flask-app` part is dumping files from the host filesystem into `/srv/flask/app`. Those new files are not affected by the previous `chown`

edit: actually I need to test that. There were potentially two issues: permissions and owner.

Comment (nrobinaubertin) [reply sample_rock/rockcraft.yaml:43]: After testing, the "owner, group" part is required.

**Review by jdkandersson — APPROVED** [review]

**Pull request merged**